<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FRDialog</class>
 <widget class="QDialog" name="FRDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="Find">
       <property name="text">
        <string>Find &amp;What:</string>
       </property>
       <property name="buddy">
        <cstring>findEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="FindSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>15</width>
         <height>36</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="findEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Replace">
       <property name="text">
        <string>&amp;Replace</string>
       </property>
       <property name="buddy">
        <cstring>replaceEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <spacer name="ReplaceSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>15</width>
         <height>36</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="replaceEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="matchCase">
     <property name="text">
      <string>Match &amp;case</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="searchBack">
     <property name="text">
      <string>Search &amp;backwards</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="close">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="replaceAllButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Replace All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="replaceButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Replace</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="findButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Find</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>findEdit</tabstop>
  <tabstop>replaceEdit</tabstop>
  <tabstop>findButton</tabstop>
  <tabstop>replaceAllButton</tabstop>
  <tabstop>close</tabstop>
  <tabstop>matchCase</tabstop>
  <tabstop>searchBack</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>replaceAllButton</sender>
   <signal>clicked()</signal>
   <receiver>FRDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>close</sender>
   <signal>clicked()</signal>
   <receiver>FRDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>32</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>signal2()</signal>
 </slots>
</ui>
